apps/web/pages/signup.tsx

```diff
@@ -161,11 +166,19 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   const ssr = await ssrInit(ctx);
   const token = z.string().optional().parse(ctx.query.token);
 
+  const isEmailVerifyEnabled = await prisma.feature.findFirst({
+    where: {
+      slug: "email-verification",
```

Comment:
**Unrelated to this PR** but I had an observation.

With this approach there is no type safety with the slugs. What if we move the features list to a file? It would be faster as well no matter we do it on client or server.

Also, we might be able to tree shake the code as well as at compile time we know the status of the feature flag.

To allow the features list to be extended by self hosters, we can have 2 configs.
 features.base.json -> Commited in git and managed by us
 features.extended.json -> Not committed in git by us but if it exists we merge it over features.base.json
 
 cc @zomars

Reply:
We're actually meant to user the useFlagsMap hook - however, this for some reason was always returning null on the signup page (works everywhere else tho)

Reply:
`useFlagsMap` is client only(requires another request). I like this approach better where you can do it on server.
I just noticed that there is typesafety through flags/config.ts, we can introduce a helper method with typesafety later.